roles/contiv_network/tasks/ovs_cleanup.yml

```
@@ -25,9 +25,16 @@
 
 - debug: var=ports
 
+- name: check selinux status
```

we can skip the task and check below in cleanup as cleanup errors are ignored.